src/managers/GuildChannelManager.js

```diff
@@ -126,6 +127,36 @@ class GuildChannelManager extends BaseManager {
     });
     return this.client.actions.ChannelCreate.handle(data).channel;
   }
+
+  /**
+   * Obtains one or more guild channels from Discord, or the channel cache if they're already available.
+   * @param {Snowflake} [id] ID of the channel
+   * @param {boolean} [cache=true] Whether to cache the new channel objects if it weren't already
+   * @param {boolean} [force=false] Whether to skip the cache check and request the API
+   * @returns {Promise<GuildChannel|Collection<Snowflake, GuildChannel>>}
```

Comment:
```suggestion
   * @returns {Promise<(?GuildChannel)|Collection<Snowflake, GuildChannel>>}
```
the function can return null here